aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/strategy/LogErrorContextMissingStrategy.java

```diff
@@ -32,6 +32,7 @@
     @Override
     public void contextMissing(String message, Class<? extends RuntimeException> exceptionClass) {
         logger.error("Suppressing AWS X-Ray context missing exception (" + exceptionClass.getSimpleName() + "): " + message);
+        logger.debug(new RuntimeException(message));
```

Comment:
The exception constructor has a lot of overhead, so we need to call it only when debug is enabled

Reply:
Thanks, a bit tired :)